microsoft-edge/origin-trials/index.md

```diff
@@ -5,13 +5,16 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 01/07/2021
+ms.date: 02/02/2023
 ---
 # Use Origin Trials in Microsoft Edge
 
+
 You can use Origin Trials to try out experimental APIs on live sites for a limited period of time.  When using Origin Trials, users of Microsoft Edge that visit your site may run code that uses experimental APIs.  To access the experimental APIs on each user machine, you don't need to go to `edge://flags` and turn on feature flags.
 
-For more information, see [Microsoft Edge Origin Trials Developer Console](https://developer.microsoft.com/microsoft-edge/origin-trials).  You can also provide feedback on the design of the API, your use cases, or your experience using the APIs to browser engineers and the web standard community.
+Instead, you register the origin (for example, `https://www.example.com:443`) that you want to use the experimental feature on and you'll be issued a token that you add to the code of your website.
+
+See the [Microsoft Edge Origin Trials Developer Console](https://developer.microsoft.com/en-us/microsoft-edge/origin-trials/) to see a list of the available origin trials and register your origin to give them a try.  You can also provide feedback on the design of the API, your use cases, or your experience using the APIs to browser engineers and the web standard community.
```

Comment:
```suggestion
To see a list of the available origin trials and register your origin to give them a try, see [Microsoft Edge Origin Trials Developer Console](https://developer.microsoft.com/microsoft-edge/origin-trials/).  You can also provide feedback to browser engineers and the web standard community about the design of the API, your use cases, or your experience using the APIs .
```
* Used pattern "To do X, see Y."
* Removed en-us/
* Pulled end of sentence 2 toward the start.  Up to you which sequence of phrases, here's an experiment.

https://developer.microsoft.com/microsoft-edge/origin-trials/